README.md

```diff
@@ -36,12 +36,12 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 2. FerretDB does not support NUL (`\0`) characters in strings.
 
-3. Collection names restrictions:
+3. Database and collection names restrictions:
 
-* collection name cannot start with the reserved prefix `_ferretdb_`.
-* collection name must not include non-latin letters, spaces, dots, dollars or dashes.
-* collection name length must be less or equal than 120 symbols.
-* collection name must not start with a number.
+* name cannot start with the reserved prefix `_ferretdb_`.
```

Comment:
We could also mention that it cannot start with capital letter
(^[**a-z**_][a-z0-9_]{0,62}$)

Reply:
> We could also mention that it cannot start with capital letter (^[**a-z**_][a-z0-9_]{0,62}$)

added.